src/main/java/com/marklogic/kafka/connect/source/RowBatcherSourceTask.java

```diff
@@ -15,10 +27,16 @@ public class RowBatcherSourceTask extends SourceTask {
 
     protected final Logger logger = LoggerFactory.getLogger(getClass());
 
+    private DatabaseClient databaseClient;
+    private DataMovementManager dataMovementManager;
+    private RowBatcher<JsonNode> rowBatcher = null;
+    private final ContentHandle<JsonNode> jsonHandle = new JacksonHandle().withFormat(Format.JSON).withMimetype("application/json");
```

Comment:
Making this a final field makes me nervous - I don't think the Java Handle classes provide any assurance that they can be reused. It's common practice to create a new one each time you need one, and they're very lightweight, so I think we're better off just creating this each time we need a new RowBatcher.